inference-engine/ie_bridges/python/src/openvino/inference_engine/CMakeLists.txt

```diff
@@ -60,13 +60,14 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     target_compile_options(${TARGET_NAME} PRIVATE "-Wno-error=register")
 endif()
 
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
```

Comment:
don't use this module. It does not support `PYTHON_EXECUTABLE` with cmake 3.13 version